services/web/server/tests/unit/with_dbs/03/folders/test_folders_groups.py

```diff
@@ -0,0 +1,4 @@
+# pylint: disable=unused-argument
+# pylint: disable=unused-variable
+# pylint: disable=too-many-arguments
+# pylint: disable=too-many-statements
```

Comment:
You could put tests that checks for NotImplementedError... or that are marked with pytest expected to fail marking ```pytest.mark.xfail(reason="not implemented")``` so that they fail when the missing implementation arrives. Also it would be a very good thing if you write the tests, and @GitHK implements the DB... then we directly see if there are missing or incomplete assumptions.

Reply:
I will be taking over some parts of the DB layer and in next PR do proper tests